cmd/mesh-bootstrap/main.go

```diff
@@ -0,0 +1,158 @@
+// +build !js
+
+// mesh-bootstrap is a separate executable for bootstrap nodes. Bootstrap nodes
+// will not share or receive any orders and its sole responsibility is to
+// facilitate peer discovery.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/0xProject/0x-mesh/keys"
+	libp2p "github.com/libp2p/go-libp2p"
+	connmgr "github.com/libp2p/go-libp2p-connmgr"
+	p2pcrypto "github.com/libp2p/go-libp2p-crypto"
+	dht "github.com/libp2p/go-libp2p-kad-dht"
+	p2pnet "github.com/libp2p/go-libp2p-net"
```

Comment:
you should use `go-libp2p-core/network` here ;-)

Reply:
We're planning to update all of our libp2p libraries in one fell swoop. Likely next week.